stablehlo/dialect/StablehloOps.td

```diff
@@ -194,7 +194,7 @@ class StableHLO_UnaryElementwiseOp<string mnemonic, list<Trait> traits,
 
 // Abs supports complex to real, so element type is not guaranteed to match.
 def StableHLO_AbsOp: StableHLO_UnaryElementwiseOp<"abs",
-    [Pure, DeclareOpInterfaceMethods<InferTypeOpInterface>],
+    [Pure, DeclareOpInterfaceMethods<InferTypeOpInterface>, HLO_MatchingOperandsAndResultScaleZeroPoint],
```

Comment:
why `abs`?
```
(C2) baseline_element_type(result) is defined as:
complex_element_type(element_type(operand)) if is_complex(operand).
baseline_element_type(operand) otherwise.
```